mapswipe_workers/mapswipe_workers/project_types/change_detection/change_detection_project.py

```diff
@@ -117,7 +117,13 @@ def validate_geometries(self):
 
             # calculate max area based on zoom level
             # for zoom level 18 this will be 5000 square kilometers
-            max_area = (20 - int(self.zoomLevel)) * (20 - int(self.zoomLevel)) * 1250
+            # max zoom level is 22
```

Comment:
this `validate_geometries` function looks like it's very similar to the one in `build_area_project.py`. Is there a way to refactor them? Maybe as a method on `BaseProject` ? and if there are only small differences, you could call `self.the_bit_that_is_different()` from `BaseProject.validate_geometries()` which would then go into the subclass. I think you would end up with a lot less code, just by looking at it quickly.

Reply:
Good comment. I will add a separate issue for this, because it will also have effects for the footprint project type. Currently we define this function for each project type. But a lot of it is redundant.

Reply:
created new issue #253